snapshot/overlay/overlay.go

```diff
@@ -236,6 +254,13 @@ func (a *activeDir) setParent(name string) error {
 }
 
 func (a *activeDir) commit(name string, c *cache) error {
+	if _, err := os.Stat(filepath.Join(a.path, "fs")); err != nil {
+		if os.IsNotExist(err) {
+			return errors.New("cannot commit view")
```

Comment:
My anticipation is that this could be replaced with an error defined in the `snapshot` package as I think this would be an error any snapshot implementation would return (trying to commit a view, an error described in the function interface definition)